Action/ConvertPaymentAction.php

```diff
@@ -41,6 +41,7 @@ public function execute($request)
             'email' => $order->getClientEmail(),
             'firstName' => isset($d['firstName']) ? $d['firstName'] : '',
             'lastName' => isset($d['lastName']) ? $d['lastName'] : '',
+            'language' => $order->getLocale() ? substr($order->getLocale(), 0, 2) : '',
```

Comment:
I've run fixer on all files from Action folder and reverted gitignore - about locale - it has to be present in storage entity. 
It can be added to documentation. I suppose that parameter has to be present. About 9.7 from documentation - if you will send wrong or absent locale - default (english) will be set. Though maybe you have better idea how to implement that parameter.